src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSource.java

```diff
@@ -343,7 +344,11 @@ private void retrievePullRequests(SCMSourceCriteria criteria, SCMHeadObserver ob
                 checkInterrupt();
                 PullRequestSCMHead head = new PullRequestSCMHead(pull.getSource().getRepository().getOwnerName(),
                         pull.getSource().getRepository().getRepositoryName(), repositoryType,
-                        pull.getSource().getBranch().getName(), pull);
+                        pull.getSource().getBranch().getName(), pull,
+                        getRepoOwner().equalsIgnoreCase(pull.getSource().getRepository().getOwnerName())
```

Comment:
So this is a bug.

It is more prominent in Bitbucket, as the user is explicitly prompted to write the fork name (although proposing the same name than the origin one as default).

What would be impact of this bug in the UI?